libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Extensions/PushNotificationManager+ActionableNotifications.swift

```diff
@@ -163,34 +163,7 @@ internal extension PushNotificationManager {
 }
 
 private extension PushNotificationManager {
-    
-    func createNotificationCategories(from type: NotificationType) -> [UNNotificationCategory] {
```

Comment:
It looks like the deleted code is duplicative of the methods in  [NotificationCategoryFactory.swift ](https://github.com/forcedotcom/SalesforceMobileSDK-iOS/blob/dev/libs/SalesforceSDKCore/SalesforceSDKCore/Classes/PushNotification/NotificationCategoryFactory.swift)

Reply:
Does `NotificationCategoryFactory.shared.createCategories` behave differently from the duplicate?